_posts/2024-09-05-releasing-apps.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: post
+title: Releasing apps at M&S
+published: true
+author: 
+  - name: Jamie Sanson
+    github: jamiesanson
+date: 2024-09-05
+categories:
+  - mobile
+tags:
+  - mobile
+  - platform
+---
+
+At Marks & Spencer we ship many apps (more than 100!) all the time. Our customer facing apps are growing by the day, and our old, time-consuming fortnightly release process was starting to show its age.
```

Comment:
I think "customer facing" is a little too internal, and given this article is predominantly about the customer apps, what about something along the lines of:

```suggestion
At Marks & Spencer we ship many apps (more than 100!) all the time. Two of those apps, the M&S flagship app on the [App Store](link...) and [Play Store](link...), have been growing by the day, and our old, time-consuming fortnightly release process was starting to show its age.
```

Let's link those too for SEO :)

Reply:
Yeah, great shout! Done [1dc2e2a](https://github.com/marksandspencer/marksandspencer.github.io/pull/18/commits/1dc2e2afb0a671b7b6f31847175691549c7c0310)